packages/debian/metalk8s-sosreport/debian/copyright

```diff
@@ -0,0 +1,12 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: metalk8s-sosreport
+Source: <https://github.com/scality/metalk8s>
+#----------------------------------------------------------------------------
+#
+# 		    GNU GENERAL PUBLIC LICENSE
```

Comment:
No I don't think we want to use `GPL` licenses. Let's stick with the same licensing policy as for the parent repo, Apache v2 (cf. [here](https://github.com/scality/metalk8s/blob/development/2.4/LICENSE)).

Reply:
Depends if SOS is GPL and requires its plugin to be GPL as well.

To be checked, but yeah if possible let's stick to a single license.

Reply:
I checked, SOS is under GPL v2 licence.
Do we keep this licence or do we keep the Apache licence ?

Reply:
This can be Apache2'd just like other parts of MetalK8s.

Reply:
I used the Apache 2 licence.
Thank you for your help about it.